submodules/wizard/wizard.js

```diff
@@ -1452,14 +1458,6 @@ define(function(require) {
 									callback: addErrorToResult(parallelCallback)
 								});
 							},
-							credit: function(parallelCallback) {
-								self.wizardRequestResourceCreateOrUpdate({
```

Comment:
Why not guard against `null` directly here instead of conditionally adding a new object prop?

Reply:
There is no good reason :sweat_smile: I'll do the guards there, it will be cleaner and easier to read :+1: